client/state/action-types.js

```diff
@@ -316,6 +316,7 @@ export const KEYRING_SERVICES_REQUEST_SUCCESS = 'KEYRING_SERVICES_REQUEST_SUCCES
 export const LAYOUT_FOCUS_SET = 'LAYOUT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_ACTIVATE = 'LAYOUT_NEXT_FOCUS_ACTIVATE';
 export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
+export const LOGIN_CLEAR_ERRORS = 'LOGIN_CLEAR_ERRORS';
```

Comment:
I think we should opt for a name that describes the action itself instead of a name that conveys how the action is going to be used. I'd like to propose something similar to the `redux-form/CHANGE` action type we used in Delphin. What about `LOGIN_FORM_UPDATE`?